sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -267,181 +271,34 @@ def plot_partial_dependence(
     >>> import matplotlib.pyplot as plt
     >>> from sklearn.datasets import make_friedman1
     >>> from sklearn.ensemble import GradientBoostingRegressor
-    >>> from sklearn.inspection import plot_partial_dependence
+    >>> from sklearn.inspection import PartialDependenceDisplay
```

Comment:
Same here. We want to revert. However, could you add `.from_estimator` in the "See Also" above.